### PR TITLE
[Bugfix] Use a special unicode as the separator in search space

### PR DESCRIPTION
**Pull request by sxjscience**
Issue #: https://github.com/awslabs/autogluon/issues/560

Description of changes:

- Change the splitter in the nested search space to a special unicode character: u'▁'  (U+2581).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-564-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-564/4/index.html

File: autogluon/core/space.py [hunk 1]
```diff
@@ -7,11 +7,15 @@
 __all__ = ['Space', 'NestedSpace', 'AutoGluonObject', 'List', 'Dict',
            'Categorical', 'Choice', 'Real', 'Int', 'Bool']
 
+SPILTTER = u'▁'  # Use U+2581 as the special symbol for splitting the space
```
Comment (Innixma) [inline autogluon/core/space.py:10]: Typo: SPLITTER

Comment (sxjscience) [reply autogluon/core/space.py]: Revised.

**Comment by szha** [comment]
Job PR-564-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-564/5/index.html

**Comment by szha** [comment]
Job PR-564-6 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-564/6/index.html

**Pull request merged**